lib/logger/lib/logger.ex

```diff
@@ -395,7 +397,11 @@ defmodule Logger do
   @spec flush :: :ok
   def flush do
     _ = GenEvent.which_handlers(:error_logger)
-    _ = GenEvent.which_handlers(Logger)
+    handlers = GenEvent.which_handlers(Logger)
+    _ = for {Logger.Backends.Console, _} = handler <- handlers do
```

Comment:
do you mean have all backends handle `:flush` in `handle_call/2`? That sounds much better.